README.md

```diff
@@ -80,27 +93,30 @@ there are several problems they still face:
   
     This disparate set of scheduling APIs makes it onerous for developers to
     write scheduling code, and requires expert knowledge of the Browser's event
-    loop to do so. Creating a unified native scheduling API can alleviate this.
-
-    This is also addressed by the [postTask API](./explainers/prioritized-post-task.md).
+    loop to do so. Creating a unified native scheduling API can alleviate this,
+    which we are also addressing with the [scheduler.postTask() API](./explainers/prioritized-post-task.md).
 
 ## Additional Scheduling Problems
 
 The problem as described above only covers part of the scheduling problem
-space.  Additionally, there are developer needs for things like detecting when
+space. Additionally, there are developer needs for things like detecting when
 a frame is pending, throttling the frame rate, and avoiding layout thrashing.
+Some of the other APIs we are considering in this space are noted [here](./misc/low-level-apis.md).
 
-Some of the other APIs we're considering in this space are noted
-[here](./misc/low-level-apis.md).
-
-## Explainer Links
+## APIs and Status
 
- * [scheduler.yield()](./explainers/yield-and-continuation.md)
- * [scheduler.postTask()](./explainers/prioritized-post-task.md)
- * [isInputPending()](https://github.com/WICG/is-input-pending)
+ | API | Abstract | Status | Links |
+ | --- | --- | --- | --- |
+ | `scheduler.postTask()` | An API for scheduling and controlling prioritizing tasks. | An origin trial recently concluded in Chrome M89, and we are currently working towards shipping the API. The feature is available [behind a flag](./origin-trial-status.md) in Chromium. | [Explainer](./explainers/prioritized-post-task.md) [Spec](https://wicg.github.io/scheduling-apis/) |
+ | `scheduler.yield()` | An API for breaking up long tasks by yielding to the browser, continuing after being rescheduled by the scheduler. | This feature is actively being designed, and the explainer will be updated soon with our current thinking. | [Explainer](./explainers/yield-and-continuation.md) |
+ | `scheduler.wait()` | This enables tasks to yield and resume after some amount of time, or perhaps after an event has occurred. | This feature is currently being co-designed with `scheduler.yield()`. | [Related Discussion](https://github.com/WICG/scheduling-apis/issues/19) |
+ | `scheduler.currentTaskSignal` | This API provides a way to get the currently running task's `TaskSignal`, which can be used to schedule dependent tasks. | This API is available behind a flag in Chromium, but we are currently re-evaluating the API based on the greater notion of a *userspace task* in context of `scheduler.yield()`. | [Explainer](./explainers/post-task-propagation.md) |
```

Comment:
"greater" is a little confusing. "Larger"? Or just "notion"? You don't have the "userspace task" essay written yet, right?